classes/patterns/class-issues.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * Issues class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Issues pattern includes:
+ * Column, Image, Heading, Paragraph, Media & Text.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Issues extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/issues';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
+			'keywords'   => [ 'column', 'image', 'heading', 'paragraph', 'media', 'text' ],
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:group {"backgroundColor":"grey-10","className":"grey-background"} -->
+				<div class="issues-block-pattern wp-block-group grey-background has-grey-10-background-color has-background block-wide">
+					<!-- wp:group -->
+					<div class="wp-block-group container">
+						<!-- wp:heading {"textAlign":"center","fontSize":"x-large"} -->
+						<h2 class="has-text-align-center has-x-large-font-size" id="we-re-fighting-to-save-our-planet">We are fighting to save our planet</h2>
+						<!-- /wp:heading -->
+						<!-- wp:paragraph {"align":"center","className":"issues-description","fontSize":"medium"} -->
+						<p class="has-text-align-center issues-description has-medium-font-size">Rising sea levels, more extreme weather events, loss of biodiversity. Our climate is breaking down and destructive industries continue to threaten our forests, oceans and air. But together we can overcome these challenges.</p>
+						<!-- /wp:paragraph -->
+						<!-- wp:group -->
+						<div class="wp-block-group">
+							<!-- wp:columns -->
+							<div class="wp-block-columns">
+								<!-- wp:column -->
+								<div class="wp-block-column">
+									<!-- wp:media-text -->
+										<div class="wp-block-media-text alignwide is-stacked-on-mobile">
+										<figure class="wp-block-media-text__media"></figure>
+										<div class="wp-block-media-text__content">
+										<!-- wp:paragraph {"placeholder":"Description","style":{"typography":{"fontSize":"17px"}}} -->
```

Comment:
We should add translation strings for placeholders, you can see [how I added them for the Highlighted CTA pattern](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/814/files#diff-5c0d4b33eaa8eb18a44ba1f1e0fb04b29c8c3492b6b4dfdc95f5dd4edeaf8c26R44)